blog/_posts/2018-05-01-extensible-broadcast-fusion.md

```diff
@@ -0,0 +1,62 @@
++---
+ +layout: post
+ +title:  "Extensible broadcast fusion"
+ +author: <a href="https://github.com/mbauman">Matt Bauman (Julia Computing)</a>
+ +---
+
+Julia version 0.7 brings with it an exciting new feature: the ability to customize broadcast fusion! This recently-merged change is the culmination of a long iterative design process that involved many members of the community. We have converged on a highly extensible interface that should satisfy many use-cases.  In this blog post I'll explain why this is a big deal by reviewing some of the key features and just scratch the surface of all that is possible with this new design. I'm quite certain that our enterprising community will come up with many more clever ways to exploit this new structure in the future.
+
+Julia now uses a first-class datastructure to "lazily" represent a fused broadcast expression before executing it. If you're not a package developer this may not mean all that much to you, but you'll still reap the many rewards. In Base Julia and its standard libraries alone, this means:
```

Comment:
Perhaps worth referencing Steven G. Johnson's post that talks about fused broadcast: https://julialang.org/blog/2017/01/moredots.